### ISSUE TITLE
sneak right clicking one of my backpacks causes a crash

### ISSUE THREAD

**Issue by 1skandar** [gr8pefish/IronBackpacks#34]
http://pastebin.com/ZSbhivVe

It has Advanced Filter, Crafting, Damage, Button, and Restocking upgrades. 

As related possible issue, I was trying to open the GUI because I thought it was picking up stuff it shouldn't. When I went to check the filter bam, crash. I would like to check the filter and confirm, but, yeah, the first bug is keeping me from verifying the second.

**Comment by gr8pefish** [comment]
Hmm I think I will need more information.

1) Can you get the server log? This client one doesn't tell me too much.
2) What version of Java are you using? Forge version?
3) What Iron Backpacks version?
4) Are you using the updated versions of NEI, inventory tweaks, etc?

**Comment by 1skandar** [comment]
Sorry, this is the latest 1.7.10 version.

And this is the crash report generated. I don't have the actual log that goes with this as I re-loaded and kept playing and that log gets overwritten.

Java 1.8.9_66 and Forge 10.13.4.1566.

And, yes, everything is up-to-date to the latest CurseForge.

**Comment by gr8pefish** [comment]
Okay, fixed this issue. 

Unfortunately, I overestimated the time I would have recently, so I made a few changes that I have to test before putting out a version with this fix. Normally I would just revert and push out a quickfix (or optimally have a branch for these dev. changes, silly gr8pefish not doing that), but since the additions aren't too massive it is just as fast to test that I implemented them correctly. If I was at home it wouldn't take long at all, but I am on the road traveling, so my work is intermittent at best.

TL;DR: I made too many features for the next branch, so this fix will not be out for a (couple) day(s), sorry :(

**Comment by gr8pefish** [comment]
Also, once I push that fix out, if you could let me know if the filter was misbehaving and picking up stuff it shouldn't I would appreciate if you made another bug report for that!

**Comment by 1skandar** [comment]
Can do. Thanks

**Comment by gr8pefish** [comment]
Forgot to only close the issue when I actually push out the fixed version.

**Comment by 1skandar** [comment]
As a comment, the backpack was actually picking things up correctly. Apparently AE likes to put everything it can into one item ID so fuzzy matching pretty much matches like 50% of the mod. I thought as much once I looked at it closer, but couldn't confirm until I could verify that I did have it fuzzy matching AE quartz.

**Comment by gr8pefish** [comment]
Cool, thanks to know. 

As a random aside, I don't have time really to actually play the game currently, so any feedback you can give me about balance/design decisions from a player perspective is very helpful.

Thanks!
